antelope_contracts/contracts/deposit_proxy/CMakeLists.txt

```diff
@@ -0,0 +1,4 @@
+add_contract( deposit_proxy deposit_proxy src/deposit_proxy.cpp )
+target_include_directories( deposit_proxy PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/include )
+target_ricardian_directory( deposit_proxy ${CMAKE_CURRENT_SOURCE_DIR}/ricardian )
```

Comment:
Does CDT's option for it to stop warning about missing Ricardians not work? We don't bother with Ricardians in this project. We just want CDT to stop spamming warnings. If we can do that without this line and without the `ricardian` directory, that would be a lot cleaner.